### PR TITLE
Show frameRate value in sample.

### PR DESCRIPTION
**Pull request by zhangyihui1**

### CODE REVIEW

File: src/sample/conference/src/main/java/owt/sample/conference/VideoFragment.java [hunk 1]
```diff
@@ -151,7 +156,14 @@ void updateStats(RTCStatsReport report, boolean outbound) {
                         lastBytesReceived = bytes;
                     }
 
-                    packetsSR = (long) members.get(outbound ? "packetsSent" : "packetsReceived");
+                    long frameSR = (long) members.get(outbound ? "framesEncoded" : "framesDecoded");
```
Comment (jianjunz) [inline src/sample/conference/src/main/java/owt/sample/conference/VideoFragment.java]: What does `frameSR` mean?

File: src/sample/conference/src/main/java/owt/sample/conference/VideoFragment.java [hunk 2]
```diff
@@ -151,7 +156,14 @@ void updateStats(RTCStatsReport report, boolean outbound) {
...
+                    if (outbound){
+                        frameRate = (frameSR - lastFrameEncoded) * 1000 / MainActivity.STATS_INTERVAL_MS;
+                        lastFrameEncoded = frameSR;
+                    }else{
```
Comment (jianjunz) [inline src/sample/conference/src/main/java/owt/sample/conference/VideoFragment.java]: Space before and after `else`.

File: src/sample/conference/src/main/java/owt/sample/conference/VideoFragment.java (hunk 1 above)
Comment (zhangyihui1) [reply src/sample/conference/src/main/java/owt/sample/conference/VideoFragment.java]: done

File: src/sample/conference/src/main/java/owt/sample/conference/VideoFragment.java (hunk 2 above)
Comment (zhangyihui1) [reply src/sample/conference/src/main/java/owt/sample/conference/VideoFragment.java]: done

File: src/sample/conference/src/main/java/owt/sample/conference/VideoFragment.java [hunk 3]
```diff
@@ -151,14 +156,25 @@ void updateStats(RTCStatsReport report, boolean outbound) {
                         lastBytesReceived = bytes;
                     }
 
-                    packetsSR = (long) members.get(outbound ? "packetsSent" : "packetsReceived");
+                    long currentFrame = (long) members.get(outbound ? "framesEncoded" : "framesDecoded");
+                    if (outbound) {
+                        frameRate = (currentFrame - lastFrameEncoded) * 1000
```
Comment (taste1981) [inline src/sample/conference/src/main/java/owt/sample/conference/VideoFragment.java]: outbound/inbound check should be done after you calculate the frameRate. Reduce un-neccessary redundant code.

Comment (zhangyihui1) [reply src/sample/conference/src/main/java/owt/sample/conference/VideoFragment.java]: done

**Pull request merged**